osbs/api.py

```diff
@@ -305,7 +306,9 @@ def _create_scratch_build(self, build_request):
             builder_img['kind'] = 'DockerImage'
             builder_img['name'] = ref
 
-        build_config_name = 'scratch-%s' % datetime.datetime.now().strftime('%Y%m%d%H%M%S')
+        build_config_name = 'scratch-%s-%s' % (
```

Comment:
`uuid.uuid1()` seems to be doing much more then `random`:

```
Generate a UUID from a host ID, sequence number, and the current time.
If 'node' is not given, getnode() is used to obtain the hardware
address.  If 'clock_seq' is given, it is used as the sequence number;
otherwise a random 14-bit sequence number is chosen.
```
Would there be any other benefits of using it?

Also: switched to using `randrange` instead of `randint`, which had the strange behavior to include the end point.